addons/utm/models/utm_mixin.py

```diff
@@ -37,7 +37,7 @@ def default_get(self, fields):
                     Model = self.env[field.comodel_name]
                     records = Model.search([('name', '=', value)], limit=1)
                     if not records:
-                        records = Model.create({'name': value})
+                        records = Model.create({'name': value, 'is_website': True})
```

Comment:
I would put the is website stuff for links in a separate commit with its own explanation, because it is a bit tricky to understand that change.